docs/root/configuration/statistics.rst

```diff
@@ -0,0 +1,45 @@
+.. _staticitics:
+
+Statistics
+==========
+
+A few statistics are emitted to report statistics system behavior:
+
+.. csv-table::
+  :header: Name, Type, Description
+  :widths: 1, 1, 2
+
+  stats.overflow, Counter, Total number of times Envoy enters a degreted state due to shortage of shared memory
```

Comment:
"Total number of times Envoy cannot allocate a statistic due to a shortage of shared memory"